DQMOffline/LumiZCounting/interface/RunLumiRangeMap.h

```diff
@@ -0,0 +1,26 @@
+#ifndef DQMOFFLINE_LUMIZCOUNTING_RUNLUMIRANGEMAP_H
+#define DQMOFFLINE_LUMIZCOUNTING_RUNLUMIRANGEMAP_H
+
+#include <string>
+#include <vector>
+#include <map>
+
+namespace baconhep
+{
+  class RunLumiRangeMap
+  {
+    public:
+      typedef std::pair<unsigned int, unsigned int>                RunLumiPairType;
+      typedef std::map<unsigned int, std::vector<RunLumiPairType>> MapType;
+      
+      RunLumiRangeMap(){}
```

Comment:
Is this code used? If not, please remove it or move to "test/" subdirectory (if for some reason you want/need to preserve it).